src/assets/stylesheets/styles.scss

```diff
@@ -108,6 +108,85 @@
   word-break: break-all;
 }
 
+// NavGlobal
+#nav-global {
+  background: color('blue-60') linear-gradient(0deg, rgba(0, 0, 0, 0) 25%, rgba(0, 0, 0, 0.2) 100%);
+  color: #FFFFFF;
+
+  // SVG logo
+  #cloud-gov-logo {
+    width: 130px;
+    @include at-media('mobile-lg') {
+      #cloud-gov-text {
+        display: inline;
+      }
+    }
+  }
+
+  .nav-global-inner {
+    align-items: center;
+    display: flex;
+    @include u-maxw('desktop-lg');
+    @include u-padding-x(2);
+    @include u-margin-x('auto');
+    @include u-font('body', 'xs');
+    @include at-media('desktop') {
+      @include u-padding-x(4);
+    }
```

Comment:
Yeah, we definitely can but I intentionally moved them to CSS once I realized I was going to have to do some custom stuff there, anyway.  I would love to talk more about where our team wants to draw the line between using the utility classes vs mixins vs just writing something like `border: 0`.  I am sometimes reluctant to add a bunch of the uswds utilities to html elements because it reminds me of inline styles.  I was trying, at minimum, to keep most stuff either using mixins or using the `units()` and `color()` functions so that we can control everything from the settings more easily.

If it's something you feel strongly about, though, I can definitely move all the mixins into the HTML instead!

Reply:
We can move forward with this for now, but let's revisit how to handle custom CSS on top of a utilities-first approach sometime soon!